addOns/pscanrules/src/main/java/org/zaproxy/zap/extension/pscanrules/TestInfoSessionIdURL.java

```diff
@@ -200,19 +215,21 @@ public void scanHttpResponseReceive(HttpMessage msg, int id, Source source) {
                 }
             }
         }
+
         if (!found && msg.getRequestHeader().getURI().getEscapedPath() != null) {
             // Handle jsessionid like:
             // http://tld.gtld/fred;jsessionid=1A530637289A03B07199A44E8D531427?foo=bar
             Matcher jsessMatcher = null;
+            String path = "";
             try {
-                jsessMatcher =
-                        Pattern.compile("jsessionid=[\\dA-Z]*", Pattern.CASE_INSENSITIVE)
-                                .matcher(msg.getRequestHeader().getURI().getPath());
+                // We lowercase the path because the regex has a specific session ID
```

Comment:
This comment contradicts the code, the pattern is case-insensitive.

Reply:
We tested it without this change and it didn't work, which is why we added this change.

Reply:
Nevermind. We retested it and it you are correct, the lowercase is not needed. We'll remove it. @thc202 - can you please respond to my other comments from 3 days ago to let us know what you think so we can get this one finished too?

Reply:
All answered.